site/content/docs/01-component-format.md

```diff
@@ -109,6 +109,20 @@ Because Svelte's reactivity is based on assignments, using array methods like `.
 </script>
 ```
 
+```sv
+<script>
+	let arr = [0, 1];
+
+	function handleClick () {
+		// this method call does not trigger an update
+		arr.push(2);
+		// this assignment will trigger an update
+		// if the markup references `arr`
```

Comment:
I wonder if this should go on the previous line? it's sort of a short line break. Probably fine either way though